data/hails.txt

```diff
@@ -8,244 +8,376 @@
+		"at the Syndicated Systems Velodrome!"
+		"at Betelgeuse Stadium!"
+		"on Hestia!"
+		"on Earth!"
+		"at Lionheart Arena!"
+		"at the Grange on Heartland!"
+		"at a resort on Vail!"
+		"at Canyonland Park!"
 
-phrase "band"
-	word
-		`"Confined Brain Of The Liberal Dork" `
-	phrase
-		"musical venue"
-	word
-		"The song "
-		"Their song "
-	word
-		`"Dude, Where's My Bible?" `
-		`"Re-Inventing Rythmn" `
-		`"Abstraction Is the Answer" `
-		`"Outrageous Activism" `
-		`"Cheap Entertainment" `
 
-phrase "band"
-	word
-		`"Catastrophe Aftermath" `
-	phrase
-		"musical venue"
+phrase "friendly civilian"
 	word
-		"The song "
-		"Their song "
+		"You should watch the video of"
+		"Tune your feed to the video of"
+		"Just watched the video of"
+		"I'm listening to"
+		"I just got tickets to see"
+		"I just heard"
+		"I just saw"
 	word
-		`"The Devil's Car" `
-		`"Selfish Thunder" `
-		`"Revenge Makes You Cry" `
-		`"Nature Of The Revolt" `
-		`"Instant Devastation" `
-
-phrase "band"
-	word
-		`"Swift Of The Sponge" `
+		" "
 	phrase
-		"musical venue"
-	word
-		"The song "
-		"Their song "
-	word
-		`"Weakness And All That" `
-		`"Trouble Will Return" `
-		`"Golden Smart Mouth" `
-		`"Burning Pain" `
-		`"Wisdom In Another World" `
-
-phrase "band"
+		"band"
 	word
-		`"Shrapnel Vacuum" `
+		" "
 	phrase
 		"musical venue"
-	word
-		"The song "
-		"Their song "
-	word
-		`"Fast Destiny" `
-		`"Fury Of My Ruggedness" `
-		`"Expensive Massacre" `
-		`"Fear Machine" `
-		`"Prisoner Of Indifference" `
 
-phrase "band"
+phrase "friendly civilian"
 	word
-		`"Daniel Suburbia" `
-	phrase
-		"musical venue"
+		"Ugh, I can't believe"
+		"I don't know why"
+		"It's a mystery why"
+		"I have no idea why"
+		"It's burning me up that"
+		"Nobody can explain to me why"
 	word
-		"The song "
-		"Their song "
+		" "
 	word
-		`"Nothing To Do With War" `
-		`"Loyalty Saved Me" `
-		`"Heaven All Over You" `
-		`"Nature Of The Heart" `
-		`"Holy Adventure" `
-
-phrase "band"
+		"the kids are"
+		"people are"
+		"anyone is"
+		"everyone's"
+		"my crew is"
+		"some of my friends are"
 	word
-		`"Volume Maneuver" `
-	phrase
-		"musical venue"
+		" still"
+		""
 	word
-		"The song "
-		"Their song "
+		" "
 	word
-		`"Loose Parade" `
-		`"The Average Girl" `
-		`"Foreign Doubt" `
-		`"Freedom Unlocked" `
-		`"Waiting For Kindness" `
-
-phrase "band"
+		"following"
+		"listening to"
+		"paying attention to"
+		"giving their money to"
+		"paying to listen to"
+		"so in love with"
 	word
-		`"Love Nugget" `
+		" "
+		" that new group "
+		" that new band "
+		" those hacks "
 	phrase
-		"musical venue"
-	word
-		"The song "
-		"Their song "
-	word
-		`"Jet Fueled Woman" `
-		`"Sin Heals A Broken Heart" `
-		`"Talent Of The Backseat" `
-		`"Beat Orgy" `
-		`"Fairground Sinner" `
-
-phrase "band"
+		"band"
 	word
-		`"Under-paid Bandit" `
-	phrase
-		"musical venue"
+		". "
 	word
-		"The song "
-		"Their song "
+		"Their music"
+	word
+		" "
```

Comment:
small tip (besides every band owning every song)
putting a space bar after "Their music" (to mke it "Their music " can save a word being only " "
don't know if you know this & it works either way, but can save some lines